src/chrome/content/rules/Sony.com.au.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	For other Sony coverage, see
+		+ Sony.xml
+
+	Nonfunctional hosts in *.sony.com.au:
+		- sony.com.au (r)
+		- content.sony.com.au (t)
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
```

Comment:
Maybe you could replace `(r)` and `(t)` at the top with `(connection refused)` and `(timeout on https)` and remove this list?